src/main/resources/application.yaml

```diff
@@ -93,4 +93,9 @@ host-cookie:
   opt-out-url: http://prebid.org/optout
   opt-in-url: http://prebid.org/optin
   ttl-days: 90
-
+gdpr:
+  default-value: 1
+  eea-countries: AT,DE,NL,BG,GR,PL,BE,HU,PT,CY,IE,RO,CZ,IT,SK,DK,LV,SI,EE,LT,ES,FI,LU,SE,FR,MT,GB,IS,NO,LI,AI,KY,AN,BQ,CW,SX,AW,FK,NC,RE,MQ,GP,GF,YT,PN,BM,PF,SH,AQ,TF,PM,IO,GL,GS,VG,TC,IC,MS,WF
+geolocation:
+  endpoints:
+    setuid: false
```

Comment:
Since the same setting will be used for turning on/off ip lookup for /cookie_sync endpoint it makes sense to rename it to something like "cookie-sync" (meaning "all endpoints related to cookie sync"). What do you think?

Reply:
ok, make sense.